src/Package/Impl/project.15.0.json

```diff
@@ -14,6 +14,7 @@
     "Microsoft.VisualStudio.Shell.Interop.10.0": "10.0.30319",
     "Microsoft.VisualStudio.Shell.Interop.11.0": "11.0.61030",
     "Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime": "15.0.25726-Preview5",
+    "Microsoft.VisualStudio.Shell.Interop.15.0.DesignTime": "15.0.25726-Preview5",
```

Comment:
Yeah, these assemblies don't include old types in newer versions. So whatever we were using from 14.0 (which I think is a different `IVsDebuggerX` interface), we're still using it, and need it.